src/registrar/management/commands/load_organization_data.py

```diff
@@ -0,0 +1,263 @@
+"""Data migration: Send domain invitations once to existing customers."""
+
+import argparse
+import json
+import logging
+import os
+
+from django.core.management import BaseCommand
+from registrar.management.commands.utility.extra_transition_domain_helper import OrganizationDataLoader
+from registrar.management.commands.utility.terminal_helper import TerminalColors, TerminalHelper
+from registrar.management.commands.utility.transition_domain_arguments import TransitionDomainArguments
+from registrar.models import TransitionDomain, DomainInformation
+from django.core.paginator import Paginator
+from typing import List
+
+from registrar.utility.errors import LoadOrganizationError, LoadOrganizationErrorCodes
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Send domain invitations once to existing customers."
+
+    def add_arguments(self, parser):
+        """Add command line arguments."""
+
+        parser.add_argument(
+            "migration_json_filename",
+            help=("A JSON file that holds the location and filenames" "of all the data files used for migrations"),
+        )
+
+        parser.add_argument("--sep", default="|", help="Delimiter character")
+
+        parser.add_argument("--debug", action=argparse.BooleanOptionalAction)
+
+        parser.add_argument("--directory", default="migrationdata", help="Desired directory")
+
+        # Serves as a domain_additional_filename override
+        parser.add_argument(
+            "--domain_additional_filename",
+            help="Defines the filename for additional domain data",
+        )
+
+        # Serves as a organization_adhoc_filename override
+        parser.add_argument(
+            "--organization_adhoc_filename",
+            help="Defines the filename for domain type adhocs",
+        )
+
+    def handle(self, migration_json_filename, **options):
+        """Process the objects in TransitionDomain."""
+
+        # === Parse JSON file === #
+        json_filepath = os.path.join(options["directory"], migration_json_filename)
+
+        # If a JSON was provided, use its values instead of defaults.
+        with open(json_filepath, "r") as jsonFile:
```

Comment:
**non blocking comment**: if you know you have a section of that does a specific task, that 100 percent should be a function. This block reads a json so a function with something like that in it's name makes sense here instead.

Reply:
Totally reasonable, I was actually thinking that too - I just wasn't sure if it was overboard